#1124 | Option to add and view Colors for Report Card removed!

ombhardwajj opened the pull request:
This PR fixes #1124  . Option to add and view colours for Report Card removed!
<img width="1423" alt="CRC" src="https://github.com/avniproject/avni-webapp/assets/115864495/24a5dcfb-6c57-4cc1-86fe-21583322f833">
<img width="1428" alt="ORC" src="https://github.com/avniproject/avni-webapp/assets/115864495/d38d964e-3e3f-472b-958d-5247fd35db13">
<img width="1438" alt="VRC" src="https://github.com/avniproject/avni-webapp/assets/115864495/ebdc23cd-f125-43da-9d6b-0b72dde601ee">

vinayvenu commented:
@ombhardwajj the issue says remove colors only for standard report cards. It looks like you have completely removed colors.

vinayvenu commented:
Look for the question "Is standard report card"

ombhardwajj commented:
<img width="1419" alt="col1" src="https://github.com/avniproject/avni-webapp/assets/115864495/adc95f58-988b-4fc9-bca4-974da14dfb37">
<img width="1439" alt="col2" src="https://github.com/avniproject/avni-webapp/assets/115864495/100540f8-7df8-4788-b4ad-ad5be059030d">


@vinayvenu , the issue says "Dont provide option to set colors" therefore If I am not wrong we had remove the colour picker. I've removed it and added back all other stuff I had removed in the last commit. Please have a look!

vinayvenu commented:
@ombhardwajj not sure if I was clear enough last time. The color and color picker are not required if and only if the user switches on the option  "Is Standard Report Card?"

ombhardwajj commented:
<img width="966" alt="rclist" src="https://github.com/avniproject/avni-webapp/assets/115864495/b137bb70-60b3-4d36-bd1b-f8e1498e3fdf">
<img width="952" alt="nstdrc" src="https://github.com/avniproject/avni-webapp/assets/115864495/a4ff6f68-39cc-4d8f-b5dc-7cc2aaf469ac">
<img width="994" alt="stdrc" src="https://github.com/avniproject/avni-webapp/assets/115864495/bceac05c-d8f2-49d4-a7f3-db3870937bcd">
I finally understood what was wanted ! I have modified the code , the option to set colors disappears for standard report cards.

vinayvenu commented:
@ombhardwajj the changes look good. Some changes before I merge this
1. Can you squash all three commits to one so it is easier to understand
2. Follow the commit message guidelines here - https://avni.readme.io/docs/commit-guidelines

ombhardwajj commented:
@vinayvenu , I have squashed all the necessary commits and added followed the commit format is #<issueNumber> | <commitMessage> in the title of this PR.

vinayvenu commented:
Merged. Thanks
